### PR TITLE
Sample Importer Attribute Cleanup

### PR DESCRIPTION
**Pull request by ebelter**
The PR mostly addresses the elimination of the passing around of entity attributes. Also removed is the updating of attributes of existing samples. WIP - We are still passing the library extension around, but that will be it's own PR.

### CODE REVIEW

File: lib/perl/Genome/Sample/Command/Import/Base.pm [hunk 1]
```diff
@@ -44,10 +44,13 @@ class Genome::Sample::Command::Import::Base {
         # source
         _individual => { is => 'Genome::Individual', is_optional => 1, },
         _individual_name => { is => 'Text', },
+        _individual_attributes => { is => 'Hash', },
```
Comment (tmooney) [inline lib/perl/Genome/Sample/Command/Import/Base.pm]: I think we more commonly use `HASH`, since that matches `ref ($cmd->_individual_attributes)`.

**Comment by ebelter** [comment]
UR should accept both. Updated.

**Comment by tmooney** [comment]
I agree; I believe it does currently work either way.  (It gets converted to `UR::Value::HASH` internally.)

+1 :barber:

**Comment by acoffman** [comment]
+1 :palm_tree:

**Comment by ebelter** [comment]
Thanks! Random emoji: :nail_care:

**Pull request merged**